features/step_definitions/registry.rb

```diff
@@ -385,8 +386,16 @@
 end
 
 Given /^default image registry route is stored in the#{OPT_SYM} clipboard$/ do |cb_name| 
+  ensure_admin_tagged
+  ensure_destructive_tagged
   org_proj_name = project(generate: false).name rescue nil
   cb_name ||= :registry_route
   cb[cb_name] = route('default-route', service('default-route',project('openshift-image-registry'))).dns(by: admin)
   project(org_proj_name) if org_proj_name
 end
+
+Given /^current generation number of#{OPT_QUOTED} deployment is stored into#{OPT_SYM} clipboard$/ do |name, cb_name|
+  ensure_admin_tagged
```

Comment:
getting `deployment` info should not require `admin` tag. Maybe for some deployments. But we will already be admins.

Reply:
Yeah, you are correct. Tag is removed.